internal/templates/components/image/metadata.templ

```diff
@@ -110,8 +112,14 @@ func ImageDateTime(viewData common.ViewData, imageIndex int) string {
 templ imageMetadata(viewData common.ViewData, imageIndex int) {
 	{{ showAlbumPersonName := (viewData.ShowAlbumName && viewData.Images[imageIndex].ImmichImage.KioskSource == kiosk.SourceAlbums) || (viewData.ShowPersonName && viewData.Images[imageIndex].ImmichImage.KioskSource == kiosk.SourcePerson) }}
 	{{ showDateTime := viewData.ShowImageDate || viewData.ShowImageTime }}
+	{{ showUser := viewData.ShowUser }}
```

Comment:
I thought you said you didn't know  Templ 😉

Reply:
yea well I didn't really know go before this project... 😶

Reply:
Oh really?! How are you finding it and navigating my (probably overly complicated) project?

Reply:
I don’t have another go project to compare it to, but it wasn’t difficult to step into compared to some of the other codebases I’ve worked on. 😉